src/core.ts

```diff
@@ -258,13 +258,19 @@ const predefinedChildrenByUid = Object.fromEntries(
 export const HIDE_REGEX = /<%HIDE%>/i;
 
 export const getCustomWorkflows = () =>
-  window.roamAlphaAPI
+  window.roamAlphaAPI.data.fast
     .q(
-      `[:find ?s ?u :where [?r :block/uid ?u] [?r :block/string ?s] [?r :block/refs ?p] (or [?p :node/title "SmartBlock"] [?p :node/title "42SmartBlock"])]`
+      `[:find
+(pull ?r [:block/uid :block/string])
+:where
+  (or
+    (and [?sb :node/title "SmartBlock"] [?r :block/refs ?sb])
+    (and [?sb :node/title "42SmartBlock"] [?r :block/refs ?sb])
+)]`
```

Comment:
the previous version broke when the user didn't have `42SmartBlock` in their graph -.-

I thought about getting rid of it, but this works too so kept it for any legacy users out there